secant/UIComponents/Chips/EnumeratedChip.swift

```diff
@@ -8,102 +8,55 @@
 import SwiftUI
 
 struct EnumeratedChip: View {
+    let basePadding: CGFloat = 14
```

Comment:
Can we add a space after this to help the property wrapper stand out and read better?  either that or change the property wrapper to be on the same line as the index, either one would improve readability.